### DISCUSSION TITLE
Allow for tablename.list in slashcommands to open up a list ?

### DISCUSSION THREAD

**Discussion by iar42** [arnoudkooi/SN-Utils#346] [Ideas]
This is probably just be me and my old habits of using tablename.list :)  but would it make sense for SN Utils to open up a list of records when _tablename_.list is written as a slashcommand?

Example:

These two would work identically and open up a list of Incidents:
/incident
/incident.list

**Comment by arnoudkooi** [comment]
Hmm, SN Utils does it differently :)

To be consistent, this would require reimplementing all variants (.LIST, .config, .form etc.)
Not really wanna do that, as all ad more are already available.
